frontend/scripts/analytics/profile-node.events.js

```diff
@@ -0,0 +1,32 @@
+import { WIDGETS__SET_ENDPOINT_DATA } from 'react-components/widgets/widgets.actions';
+import { PROFILE_NODE__ON_PDF_DOWNLOAD } from 'react-components/profile-node/profile-node.actions';
+import { GET_NODE_SUMMARY_URL } from 'utils/getURLFromParams';
+
+export default [
+  {
+    type: WIDGETS__SET_ENDPOINT_DATA,
+    action: 'Profile page visited',
+    category: 'profile',
+    getPayload(action, state) {
+      const { data } = action.payload;
+      const {
+        location: { query },
+        app: { contexts }
+      } = state;
+      const context = contexts.find(ctx => Number(query.contextId) === ctx.id);
+
+      // eslint-disable-next-line
+      return `${data?.jurisdiction_name} - ${data?.column_name} - ${context?.countryName} - ${
+        context.commodityName
+      } - ${query?.year}`;
+    },
+    shouldSend: action => action.payload.endpoint === GET_NODE_SUMMARY_URL
+  },
+  {
+    type: PROFILE_NODE__ON_PDF_DOWNLOAD,
+    action: 'Download PDF',
```

Comment:
Could we add the PROFILE category? I'm not quite sure what is that [WHERE?] on the spreadsheet

Reply:
yes I should have added that